src/desktop/analytics/main_layout.ts

```diff
@@ -20,6 +20,7 @@ const excludedRoutes = [
   "/collect(.*)",
   "/collection(.*)",
   "/collections(.*)",
+  "/fair(.*)",
```

Comment:
@anandaroop we're not currently using the `/fair` route for anything in production (excluding our new page), right?

I want to make sure I don't break any existing page view tracking with this change.

Reply:
Correct, as of now. I'll PR the `/fair2` → `/fair` change this morning.